drivers/flash/flash_stm32_qspi.c

```diff
@@ -503,7 +745,11 @@ void HAL_QSPI_RxCpltCallback(QSPI_HandleTypeDef *hqspi)
 /*
  * Tx Transfer completed callback.
  */
-void HAL_QSPI_TxCpltCallback(QSPI_HandleTypeDef *hqspi)
+#ifndef STM32_OSPI_MIMIC_QSPI
+void HAL_QSPI_TxCpltCallback(QSPI_HandleTypeDef * hqspi)
+#else
+void HAL_OSPI_TxCpltCallback(OSPI_HandleTypeDef *hqspi)
```

Comment:
Since these methods are defined in a header file, they shouldn't be renamed right?